portfolio/src/main/webapp/script.js

```diff
@@ -274,13 +282,39 @@ function handleSubmit(e) {
     success: () => $('.comment').length > 0 ? loadComments(LoadType.RELOAD) : loadComments(LoadType.LOAD),
   });
   $(this).find('input,textarea').val('');
+  $("#anonCheck").prop("checked", false);
+  $('#displayedName').prop('required', true);
 }
 
 $('#rec-form').submit(handleSubmit);
+$('#anonCheck-container').click(() => {
+  $('#displayedName').prop('required', !$('#anonCheck').is(':checked'));
```

Comment:
maybe we can keep the pattern of using prop
`$('#displayedName').prop('required', !$('#anonCheck').prop('checked'));`